GramAddict.py

```diff
@@ -110,6 +110,10 @@ def main():
         session_state = SessionState()
         session_state.args = args.__dict__
         sessions.append(session_state)
+        if args.screen_care == "on":
```

Comment:
The rest of the args use `-`, we should keep it uniform. Also, is there a better name we could use over screen care? Maybe screen-sleep  or screen-off - something that better defines what it does.

Reply:
it's screen-sleep now